cli/script/command-executor.ts

```diff
@@ -1174,6 +1256,69 @@ function release(command: cli.IReleaseCommand): Promise<void> {
         });
 }
 
+export var releaseReact = (command: cli.IReleaseReactCommand): Promise<void> => {
+    var platform: string = command.platform.toLowerCase();
+    var entryFile: string = command.entryFile;
+    var outputFolder: string = path.join(getTmpDir(), "CodePush");
+    var releaseCommand: cli.IReleaseCommand = <any>command;
+    releaseCommand.package = outputFolder;
+    
+    if (platform !== "ios" && platform !== "android") {
+        throw new Error("Platform must be either \"ios\" or \"android\".");
+    }
+    
+    try {
+        var projectPackageJson = require(path.join(process.cwd(), "package.json"));
+        var projectName: string = projectPackageJson.name;
+        if (!projectName) {
+            throw new Error("The \"package.json\" file in the CWD does not have the \"name\" field set.");
+        }
+        
+        if (!projectPackageJson.dependencies["react-native"]) {
+            throw new Error("The project in the CWD is not a React Native project.");
+        }
+    } catch (error) {
+        throw new Error("Unable to find or read \"package.json\" in the CWD. The \"release-react\" command must be executed in a React Native project folder.");
+    }
+    
+    if (!entryFile) {
+        entryFile = `index.${platform}.js`;
+        if (fileDoesNotExistOrIsDirectory(entryFile)) {
+            entryFile = "index.js";
+        }
+        
+        if (fileDoesNotExistOrIsDirectory(entryFile)) {
+            throw new Error(`Entry file "index.${platform}.js" or "index.js" does not exist.`);
+        }
+    } else {
+        if (fileDoesNotExistOrIsDirectory(entryFile)) {
+            throw new Error(`Entry file "${entryFile}" does not exist.`);
+        }
+    }
+    
+    // This is needed to clear the react native bundler cache:
+    // https://github.com/facebook/react-native/issues/4289
+    execSync("rm -rf $TMPDIR/react-*");
```

Comment:
Shouldn't we use the rimraf library that you added a dependency for? Would this work correctly on Windows?